src/types.ts

```diff
@@ -497,21 +479,24 @@ export interface DbPayment {
   totalFees: number;
 }
 
+export interface Payment extends Omit<DbPayment, "id"> {
+  id: number;
+}
+
+export enum PermissionMethodNostr {
+  NOSTR_SIGNMESSAGE = "nostr/signMessage",
+}
+
 export interface DbPermission {
   id?: number;
   createdAt: string;
   allowanceId: number;
   host: string;
-  method: string;
+  method: string | PermissionMethodNostr;
```

Comment:
hmmm... so why string and this?

Reply:
We currently do not knwo the webln-methods but we do know the nostr ones

Reply:
why don't we know the webln ones? 

or rather there are actually none that use the permissions.

Reply:
Right, so, I mean, at least currently we do not know them. At least in the context of this PR.  
Eventually we should be able to know them so we can also translate them etc.

Reply:
currently it is only what is in `PermissionMethodNostr` we know that and we change this in this PR.
why do we need to keep the string?

Reply:
Otherwise TS will fail for this: https://github.com/getAlby/lightning-browser-extension/blob/master/src/extension/background-script/actions/ln/request.ts#L39